web-client/src/components/PhoneNumber/PhoneNumber.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Input, Typography } from 'antd';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import logo from '../../assets/logo.png';
+import TitleWithAddon from '../TitleWithAddon/TitleWithAddon';
+
+const { Text } = Typography;
+
+const StyledIntro = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 50px 50px;
+`;
+
+const Logo = styled.img`
+  height: 125px;
+  width: 125px;
+`;
+
+const Description = styled(Text)`
+  margin-top: 3rem;
+`;
+
+const StyledInput = styled(Input)`
+  margin-top: 1rem;
+  width: 11rem;
+`;
+
+const Info = styled(Text)`
+  color: #ddd;
+  /* margin-top: 40px; */
+  text-align: center;
+`;
+
+const StyledButton = styled(Button)`
+  margin-top: 40px;
+`;
```

Comment:
You should try to make these styles more mobile-friendly. **Tip:** try it into an iPhone 6/7/8 and you will see what I'm talking about.

Reply:
Btw, it's a problem induced by CenteredCard

Reply:
Also, the text isn't supposed to be left-aligned after screen shrink

Reply:
> 
> 
> Btw, it's a problem induced by CenteredCard

I will fix this in another PR